src/main/java/Noisy/Event.java

```diff
@@ -0,0 +1,27 @@
+package Noisy;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private LocalDate startDate;
+    private LocalDate endDate;
+
+    public Event(String description, boolean isDone, LocalDate startDate, LocalDate endDate) {
+        super(description, isDone);
+        this.startDate = startDate;
+        this.endDate = endDate;
+    }
+
+    @Override
+    public String formatTask() {
```

Comment:
I like how you name methods as a verb that reflect what the method does, which comply with the coding standard.